contracts/lifecycle/DestructibleDelayed.sol

```diff
@@ -0,0 +1,74 @@
+pragma solidity ^ 0.4.18;
+
+import "../ownership/Ownable.sol";
+
+/**
+ * @title DestructibleDelayed
+ * @dev Contract that can be selfdestructed after a delay period.
+ */
+contract DestructibleDelayed is Ownable {
+
+   // Whether a request for the self-destruction of the contract was requested.
+   bool public destructionRequested = false;
+
+   // When self-destruction is possible
+   uint256 public destructionTime = 2**256-1;
+
+   // Minimum delay after which destroy() can be called once requested
+   uint256 public SELFDESTRUCTION_DELAY; 
```

Comment:
Any particular reason to use uppercase snake case for this state variable? Solidity's [style guide](https://solidity.readthedocs.io/en/v0.4.24/style-guide.html#constants) only recommends this for constant values, which this is not.

Reply:
Hm, I just noted this is _sort of_ a constant value, since it can only be set in the constructor. I'd make it `private` then (and provide a getter), so that contracts inheriting from this one cannot modify it.

Reply:
Yeah, it's sort-of a constant. Made private + getter.